### PR TITLE
[iluvatar] update iluvatar bert_hf

### PR DESCRIPTION
**Pull request by forestlee95**
[iluvatar_bert_hf_BI-V100_2x8.zip](https://github.com/FlagOpen/FlagPerf/files/13783034/iluvatar_bert_hf_BI-V100_2x8.zip)
[iluvatar_bert_hf_BI-V100_1x8.zip](https://github.com/FlagOpen/FlagPerf/files/13783035/iluvatar_bert_hf_BI-V100_1x8.zip)
[iluvatar_bert_hf_BI-V100_1x1.zip](https://github.com/FlagOpen/FlagPerf/files/13783036/iluvatar_bert_hf_BI-V100_1x1.zip)
[iluvatar_bert_hf_BI-V100_1x8_amp.zip](https://github.com/FlagOpen/FlagPerf/files/13790695/iluvatar_bert_hf_BI-V100_1x8_amp.zip)

### CODE REVIEW

**Comment by yuzhou03** [comment]
test_conf.py中，请添加bert-hf的天数卡运行实例

**Review by upvenly — APPROVED** [review]

File: training/iluvatar/bert_hf-pytorch/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| **计算吞吐量** | **p_core,见“性能指标”** | 不包含数据IO部分的耗时                      |
+| 训练结果       | acc,见“性能指标”        | masked_lm任务准确率(实际/目标)              |
+
+
+* 性能指标
+
+| 配置               | precision | fix_hp | e2e_time | p_whole | p_train | p_core | acc         | mem       |
+| ------------------ | --------- | ------ | -------- | ------- | ------- | ------ | ----------- | --------- |
+| BI-V100单机8卡（1x8） | fp32      | bs=12  | 11199.1   | 53.8   | 55.6   | 55.8  | 0.657/0.655 | 30.4/32.0 |
+| BI-V100两机8卡（1x1） | fp32      | bs=12  | /        |    /     |   /      | 8.0      | /           |  28.0/32.0         |
```
Comment (yuzhou03) [inline training/iluvatar/bert_hf-pytorch/README.md:42]: 两机8卡（2x8）

Comment (yuzhou03) [inline training/iluvatar/bert_hf-pytorch/README.md:42]: BI-V100单机单卡（1x1）

**Comment by yuzhou03** [comment]
rebuild image && start container 2x8 【OK】
image
image

**Review by yuzhou03 — APPROVED** [review]

**Review by shh2000 — APPROVED** [review]

**Pull request merged**